boxes/docker-compose.yml

```diff
@@ -23,6 +23,10 @@ services:
         while ! nc -z aztec 8080; do sleep 1; done;
         yarn workspace @aztec/$BOX test
       '
+    ipc: host
```

Comment:
This is fishy. What are we adding this stuff for?

Reply:
Indeed it doesn’t look amazing but it’s recommended in the playwright docs. Added it when I was debugging. I haven’t tried without it now that it runs fine, so ill give it a try